soapui/src/main/java/com/eviware/soapui/support/ModelItemNamer.java

```diff
@@ -58,4 +58,8 @@ public static String createName(String baseName, List<? extends ModelItem> model
 
         return baseName + " " + (++maxExistingIndex);
     }
+
+    private static String getName(Object nameHolder) {
+        return nameHolder instanceof ModelItem ? ((ModelItem) nameHolder).getName() : nameHolder.toString();
```

Comment:
I copied that code from Ready! API without any changes. Do we have any policy on keeping the two codebases alike (as some plugins are supposed to work in both products)?